actions/publish-dockers/action.yaml

```diff
@@ -34,6 +34,15 @@ runs:
         username: ${{ inputs.dockerhub_username }}
         password: ${{ inputs.dockerhub_token }}
 
+
+    - name: Pull the main branch of llama-stack
+      if: startsWith(github.event.inputs.version, 'nightly-')
+      uses: actions/checkout@v2
```

Comment:
Why v2 instead of v4 here?

Reply:
no reason. Just had used v2 based on previous usage. Can bump if required

Reply:
I saw you used `v4` elsewhere in the workflow thus the ask - thanks for bumping!